update import docs (#1495)
cocos-creator/creator-docs: en/release-notes/upgrade-guide-v3.0.md

Although **we do not recommend projects under development, especially projects that are about to go live, to upgrade to v3.0**, there will be a v2.x resource migration tool when v3.0 is officially released. This tool will support migrating old projects, project resources, and project code very well. Code-assisted migration will convert **JavaScript** into **TypeScript**, and automatically add component type declarations, attribute declarations and function declarations. The references of components in the scene will be preserved, and the code inside the function will be migrated in the form of comments, which can reduce the difficulty of upgrading.

Developers only need to click **File -> Import Cocos Creator 2.x project** in the main menu.

![import-menu](import-menu.png)

Next, select the root directory of the Cocos Creator 2.x project in the file browse dialog that pops up.

![import-select-project](import-select-project.png)

> > **Note**: It is recommended to upgrade to Cocos Creator 2.4.3 or above separately before re-importing; otherwise the import results cannot be guaranteed to be correct.

Reviewer: ```suggestion
> **Note**: it is recommended to upgrade to Cocos Creator 2.4.3 or above before importing to Cocos Creator 3.0 for older projects, otherwise the correctness of the import result cannot be ensured.
```

> All the resources in the Cocos Creator 2.x project will be automatically presented in the **Import Cocos Creator 2.x** project pop-up panel, where the developer can reconfirm the resources to be imported and then click the **Import** button in the bottom right corner of the panel to complete the import. If the developer wants to switch between imported 2.x projects, click on the search icon button in the image below to reselect the project.

Reviewer: ```suggestion
All the resources in the Cocos Creator 2.x project will be automatically presented in the popup **Import Cocos Creator 2.x Project** panel. Developers can reconfirm the resources to be imported and then click the **Import** button in the bottom right corner of the panel to complete the import. If the developer wants to switch the imported 2.x project, click the search icon button in the image below to reselect the project.
```

![import-project](import-panel.png)

> The **Manual** button in the bottom left corner of the panel will take you to the GitHub repository where the project plugins are imported, to [update imported plugins](https://github.com/cocos-creator/plugin-import-2.x/blob/main/README.md) or to submit feedback.

Reviewer: ```suggestion
The **Manual** button in the bottom left corner of the panel will take you to the GitHub repository for the Import Plugin, which can be used to [update the Import Plugin](https://github.com/cocos-creator/plugin-import-2.x/blob/main/README.md) or submit feedback.
```
